girder/models/user.py

```diff
@@ -46,6 +47,11 @@ def initialize(self):
         self.exposeFields(level=AccessType.ADMIN, fields=(
             'size', 'email', 'groups', 'groupInvites'))
 
+        events.bind('model.user.save.created', 'grantSelfAccess',
+                    self._grantSelfAccess)
+        events.bind('model.user.save.created', 'addDefaultFolders',
+                    self._addDefaultFolders)
```

Comment:
@zachmullen If you have a clear idea of what you'd like to see for names in ``constants``, please feel free to add a commit on top of this PR. I don't have any real opinion on this matter.

Otherwise, I can try to do it myself, but I don't yet have a clear vision for what I'd like the particular organization to be.